### ISSUE TITLE
[Accessibility][ConfigPanelText] Color alone conveys information without additional shapes, patterns, or images

### ISSUE THREAD

**Issue by jmbuss** [awslabs/iot-app-kit#2547]
Issue 16.5 [Color alone conveys information without additional shapes, patterns, or images](https://www.tpgarc.com/Analytics/Dashboard/EngagementComponentPageAssertions/1822782/Engagement/1759/ScanLog/106158/Assertion/754/AccessibilityStandard/WCAG21) 
 
 
Assertion ID: 200290
 
Target Count: 3
 
Standard: 1.4.1 Use of Color (A)
 
Affected Groups: Cognitively Impaired, Low Vision
 
Severity: Medium
 
Priority: 3
 
 
**Notes and Advice** 
The only visual means of conveying information for distinguishing visual elements when focused is with color, this is the case for the size control. This may not be perceivable to color-blind and low-vision users, nor be conveyed to users of assistive technology.

**Recommendation** 
Complement the use of color with additional visual cues, such as shapes or patterns, or text, to convey information.
 
In the case of focus indicators we recommend a clearly visible outline or border. Alternately, using a color change alone as a focus indicator can pass [1.4.1 Use of Color](https://www.w3.org/TR/WCAG/#use-of-color), provided that the difference between the focused state and the unfocused state has a contrast ratio of 3:1 or greater. For example, if the focused state of a button is indicated by a change in background color, then the difference between the focused and unfocused backgrounds must have a contrast ratio of 3:1 or greater. High-contrast color inversion (such as black to white) is usually sufficient.
 Resources 
[Well Color Us Surprised—This SC Can Be a Tricky Customer](https://www.tpgi.com/well-color-us-surprised-this-sc-can-be-a-tricky-customer/)
[Understanding Success Criterion 1.4.1: Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)

<img width="497" alt="image" src="https://github.com/awslabs/iot-app-kit/assets/107281089/ccc22bcf-2c4e-421a-88f3-25462d98724b">

<img width="501" alt="image" src="https://github.com/awslabs/iot-app-kit/assets/107281089/62796a11-9d2e-47dc-b43e-c7885833bf11">

Color contrast ratio must be 3:1 for the focus state of these buttons.
